python/cuxfilter/charts/core/core_chart.py

```diff
@@ -110,6 +129,80 @@ def _repr_mimebundle_(self, include=None, exclude=None):
                 return view.pprint()
         return None
 
+    def _to_xaxis_type(self, dates):
+        """
+        Description: convert to int64 if self.x_dtype is of type datetime
+        -----------------------------------------------------------------
+        Input:
+            dates: cudf.Series | list | tuple
+        """
+        # self.x_dtype is a computed read-only property
+        return dt.to_int64_if_datetime(dates, self.x_dtype)
+
+    def _to_yaxis_type(self, dates):
+        """
+        Description: convert to int64 if self.y_dtype is of type datetime
+        -----------------------------------------------------------------
+        Input:
+            dates: cudf.Series | list | tuple
+        """
+        # self.x_dtype is a computed read-only property
+        return dt.to_int64_if_datetime(dates, self.y_dtype)
+
+    def _xaxis_dt_transform(self, dates):
+        """
+        Description: convert to datetime64 if self.x_dtype is of type datetime
+        -----------------------------------------------------------------
+        Input:
+            dates: list | tuple of integer timestamps objects
+        """
+        # self.x_dtype is a computed read-only property
+        return dt.to_dt_if_datetime(dates, self.x_dtype)
+
+    def _yaxis_dt_transform(self, dates):
+        """
+        Description: convert to datetime64 if self.y_dtype is of type datetime
+        -----------------------------------------------------------------
+        Input:
+            dates: list | tuple of integer timestamps objects
+        """
+        # self.x_dtype is a computed read-only property
```

Comment:
should be `y_dtype` in the comment but I don't think these comments are especially necessary either.

Reply:
fixed